site/docs/master/api.md

```diff
@@ -0,0 +1,4 @@
+# Contour API Reference
+
```

Comment:
Include something about how this is generated, which commands are used, etc.

Reply:
Do you mean, how does a developer re-generate this?

Reply:
There's a documented makefile target that generates these docs. If we find later that we need more info, I'll be happy to add it in a separate PR.